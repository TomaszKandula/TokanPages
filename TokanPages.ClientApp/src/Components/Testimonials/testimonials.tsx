import * as React from "react";
import { useSelector } from "react-redux";
import { ApplicationState } from "../../Store/Configuration";
import { TestimonialsView } from "./View/testimonialsView";

export const Testimonials = (): JSX.Element => {
    const testimonials = useSelector((state: ApplicationState) => state.contentTestimonials);

    const [hasTestimonialOne, setTestimonialOne] = React.useState(false);
    const [hasTestimonialTwo, setTestimonialTwo] = React.useState(false);
    const [hasTestimonialThree, setTestimonialThree] = React.useState(false);

    const buttonTestimonialOne = React.useCallback(() => {
        setTestimonialOne(!hasTestimonialOne);
    }, [hasTestimonialOne]);

    const buttonTestimonialTwo = React.useCallback(() => {
        setTestimonialTwo(!hasTestimonialTwo);
    }, [hasTestimonialTwo]);

    const buttonTestimonialThree = React.useCallback(() => {
        setTestimonialThree(!hasTestimonialThree);
    }, [hasTestimonialThree]);

    return (
        <TestimonialsView 
            isLoading={testimonials?.isLoading}
            hasTestimonialOne={hasTestimonialOne}
            hasTestimonialTwo={hasTestimonialTwo}
            hasTestimonialThree={hasTestimonialThree}
            buttonTestimonialOne={buttonTestimonialOne}
            buttonTestimonialTwo={buttonTestimonialTwo}
            buttonTestimonialThree={buttonTestimonialThree}
            caption={testimonials?.content?.caption}
            subtitle={testimonials?.content?.subtitle}
            photo1={testimonials?.content?.photo1}
            name1={testimonials?.content?.name1}
            occupation1={testimonials?.content?.occupation1}
            text1={testimonials?.content?.text1}
            photo2={testimonials?.content?.photo2}
            name2={testimonials?.content?.name2}
            occupation2={testimonials?.content?.occupation2}
            text2={testimonials?.content?.text2}
            photo3={testimonials?.content?.photo3}
            name3={testimonials?.content?.name3}
            occupation3={testimonials?.content?.occupation3}
            text3={testimonials?.content?.text3}
        />
    );
}
