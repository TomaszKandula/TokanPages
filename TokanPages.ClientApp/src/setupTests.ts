// jest-dom adds custom jest matchers for asserting on DOM nodes.
// allows you to do things like:
// expect(element).toHaveTextContent(/react/i)
// learn more: https://github.com/testing-library/jest-dom
import "@testing-library/jest-dom";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import enableHooks from "jest-react-hooks-shallow";

enableHooks(jest);

jest.mock("react-redux", () => (
{
    useSelector: jest.fn(),
    useDispatch: jest.fn()
}));

jest.mock("connected-react-router", () => (
{
    connectRouter: jest.fn(),
    routerMiddleware: jest.fn()
}));

Enzyme.configure({ adapter: new Adapter(), disableLifecycleMethods: false });

// environmental variables used during all tests
process.env.REACT_APP_API_VER="1";
process.env.REACT_APP_BACKEND="http://localhost:5000";
