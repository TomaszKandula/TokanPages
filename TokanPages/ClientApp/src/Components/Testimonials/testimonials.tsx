import * as React from "react";
import Container from "@material-ui/core/Container";
import Box from "@material-ui/core/Box";
import Typography from "@material-ui/core/Typography";
import { Carousel } from "react-responsive-carousel";
import { ITestimonials } from "../../Api/Models";
import { renderImage } from "../../Shared/Components/renderImage";
import useStyles from "./styledTestimonials";
import "react-responsive-carousel/lib/styles/carousel.min.css";
import "./overrideCarousel.css";

export default function Testimonials(props: ITestimonials)
{
    const classes = useStyles();
    return(
        <section className={classes.section}>
            <Container maxWidth="lg"> 
                <div data-aos="fade-up">
                    <Box pt={8} pb={10}>
                        <Container maxWidth="sm">
                            <Box textAlign="center" mb={5}>
                                <Typography variant="h4" component="h2" gutterBottom={true}>
                                    {props.content.caption}
                                </Typography>
                                <Typography variant="subtitle1" color="textSecondary">
                                    {props.content.subtitle}
                                </Typography>
                            </Box>
                        </Container>
                        <Carousel showArrows={true} infiniteLoop={true} showThumbs={false} showStatus={false} autoPlay={true} interval={6100}>
                            <Box className={classes.boxPadding}>
                                {renderImage("", props.content.photo1, classes.img)}
                                <Typography variant="h4" component="h3" align="center" className={classes.title}>
                                    {props.content.name1}
                                </Typography>
                                <Typography variant="subtitle1" component="h4" align="center" className={classes.subtitle}>
                                    {props.content.occupation1}
                                </Typography>
                                <Typography variant="body1" component="p" align="center" color="textSecondary" className={classes.commendation}>
                                    {props.content.text1}
                                </Typography>
                            </Box>
                            <Box className={classes.boxPadding}>
                                {renderImage("", props.content.photo2, classes.img)}
                                <Typography variant="h4" component="h3" align="center" className={classes.title}>
                                    {props.content.name2}
                                </Typography>
                                <Typography variant="subtitle1" component="h4" align="center" className={classes.subtitle}>
                                    {props.content.occupation2}
                                </Typography>
                                <Typography variant="body1" component="p" align="center" color="textSecondary" className={classes.commendation}>
                                    {props.content.text2}
                                </Typography>
                            </Box>
                            <Box className={classes.boxPadding}>
                                {renderImage("", props.content.photo3, classes.img)}
                                <Typography variant="h4" component="h3" align="center" className={classes.title}>
                                    {props.content.name3}
                                </Typography>
                                <Typography variant="subtitle1" component="h4" align="center" className={classes.subtitle}>
                                    {props.content.occupation3}
                                </Typography>
                                <Typography variant="body1" component="p" align="center" color="textSecondary" className={classes.commendation}>
                                    {props.content.text3}
                                </Typography>
                            </Box>
                        </Carousel>
                    </Box>
                </div>        
            </Container>
        </section>
    );
}
