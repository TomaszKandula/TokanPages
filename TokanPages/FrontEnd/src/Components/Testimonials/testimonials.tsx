import * as React from "react";
import Container from "@material-ui/core/Container";
import Box from "@material-ui/core/Box";
import Typography from "@material-ui/core/Typography";
import { Carousel } from "react-responsive-carousel";
import useStyles from "./styledTestimonials";
import "react-responsive-carousel/lib/styles/carousel.min.css";
import "./overrideCarousel.css";

export default function Testimonials()
{

    const classes = useStyles();
    const content = 
    {
        cpation: "Testimonials",
        subtitle: "Here you can read few commendations, a word from people I worked with.",
        photo1: "https://maindbstorage.blob.core.windows.net/tokanpages/images/section_testimonials/joanna.jpg",
        name1: "Joanna Strzyzewska",
        occupation1: "Senior Digital Tax Specialist",
        text1: `Tomek is very motivated, has wide IT knowledge and is constantly developing in new technologies. 
                I have seen many examples of his talent. He is also a good and friendly co-worker and colleague. 
                I can recommend Tomek not only because of his technical and programming skillset but also
                because of his attitude towards changes and best practices.`,
        photo2: "https://maindbstorage.blob.core.windows.net/tokanpages/images/section_testimonials/adama.jpg",
        name2: "Adama Sow",
        occupation2: "Full-stack Developer chez DFDS POLSKA",
        text2: `Tomek have done very good work in NET Core, Azure. He is really helpful. I would recommend him.`,
        photo3: "https://maindbstorage.blob.core.windows.net/tokanpages/images/section_testimonials/scott.jpg",
        name3: "Scott Lumsden",
        occupation3: "Customer Service | Contact Center | Vendor Manager | BPO",
        text3: `Tomasz is one of those rare talents, he is extremely adaptable, and capable in turning ideas to reality. 
                His programming ability allows him to build scalable robust digital solutions.
                While working with him at the accounting shared service center for DFDS, he single handily was also able to 
                produce software solutions with ERP integration helping the AR, AP and GL departments work more effectively.`
    };

    return(
        <section className={classes.section}>
            <Container maxWidth="lg"> 
                <div data-aos="fade-up">
                    <Box pt={8} pb={10}>
                        <Container maxWidth="sm">
                            <Box textAlign="center" mb={5}>
                                <Typography variant="h4" component="h2" gutterBottom={true}>
                                    {content.cpation}
                                </Typography>
                                <Typography variant="subtitle1" color="textSecondary">
                                    {content.subtitle}
                                </Typography>
                            </Box>
                        </Container>
                        <Carousel showArrows={true} infiniteLoop={true} showThumbs={false} showStatus={false} autoPlay={true} interval={6100}>
                            <Box className={classes.boxPadding}>
                                <img src={content.photo1} className={classes.img} alt="" />
                                <Typography variant="h4" component="h3" align="center" className={classes.title}>
                                    {content.name1}
                                </Typography>
                                <Typography variant="subtitle1" component="h4" align="center" className={classes.subtitle}>
                                    {content.occupation1}
                                </Typography>
                                <Typography variant="body1" component="p" align="center" color="textSecondary" className={classes.commendation}>
                                    {content.text1}
                                </Typography>
                            </Box>
                            <Box className={classes.boxPadding}>
                                <img src={content.photo2} className={classes.img} alt="" />
                                <Typography variant="h4" component="h3" align="center" className={classes.title}>
                                    {content.name2}
                                </Typography>
                                <Typography variant="subtitle1" component="h4" align="center" className={classes.subtitle}>
                                    {content.occupation2}
                                </Typography>
                                <Typography variant="body1" component="p" align="center" color="textSecondary" className={classes.commendation}>
                                    {content.text2}
                                </Typography>
                            </Box>
                            <Box className={classes.boxPadding}>
                                <img src={content.photo3} className={classes.img} alt="" />
                                <Typography variant="h4" component="h3" align="center" className={classes.title}>
                                    {content.name3}
                                </Typography>
                                <Typography variant="subtitle1" component="h4" align="center" className={classes.subtitle}>
                                    {content.occupation3}
                                </Typography>
                                <Typography variant="body1" component="p" align="center" color="textSecondary" className={classes.commendation}>
                                    {content.text3}
                                </Typography>
                            </Box>
                        </Carousel>
                    </Box>
                </div>        
            </Container>
        </section>
    );

}
