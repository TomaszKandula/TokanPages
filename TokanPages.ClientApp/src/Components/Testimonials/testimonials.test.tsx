import "../../setupTests";
import React from "react";
import * as Redux from "react-redux";
import { render } from "@testing-library/react";
import { Testimonials } from "./testimonials";

jest.mock("react-redux", () => ({
    ...jest.requireActual("react-redux"),
    useSelector: jest.fn(),
}));

describe("test component: testimonials", () => {
    const testContent = {
        language: "eng",
        caption: "Testimonials",
        subtitle: "You can read few commendations...",
        photo1: "joanna.webp",
        name1: "Joanna",
        occupation1: "Senior Digital Tax Specialist",
        text1: "Joe is very motivated...",
        photo2: "adama.webp",
        name2: "Adama",
        occupation2: "Full-stack Developer chez DFDS POLSKA",
        text2: "Joe have done very good work...",
        photo3: "scott.webp",
        name3: "Scott",
        occupation3: "BPO",
        text3: "Joe is one of those rare talents...",
    };

    beforeEach(() => {
        jest.spyOn(Redux, "useSelector").mockReturnValueOnce({
            isLoading: false,
            content: testContent,
        });
    });

    it("should render correctly '<Testimonials />' when content is loaded.", () => {
        const html = render(<Testimonials />);
        expect(html).toMatchSnapshot();
    });
});
