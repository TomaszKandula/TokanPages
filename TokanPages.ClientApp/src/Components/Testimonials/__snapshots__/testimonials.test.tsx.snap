// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component: testimonials should render correctly '<Testimonials />' when content is loaded. 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="section "
      >
        <div
          class="MuiContainer-root container-super-wide MuiContainer-maxWidthLg"
        >
          <div
            class="text-centre pt-64 pb-120"
          >
            <div
              data-aos="fade-down"
              data-aos-delay="0"
            >
              <h1
                class="testimonials-caption-text"
              >
                TESTIMONIALS
              </h1>
            </div>
          </div>
          <div
            class="text-centre pb-120"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-6"
            >
              <div
                class="MuiGrid-root testimonials-card-holder MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4"
              >
                <div
                  data-aos="fade-up"
                  data-aos-delay="350"
                >
                  <div
                    class="MuiPaper-root MuiCard-root testimonials-card MuiPaper-elevation0 MuiPaper-rounded"
                  >
                    <img
                      alt="Testimonail photo 1 of 3"
                      class="MuiCardMedia-root testimonials-card-image MuiCardMedia-media MuiCardMedia-img"
                      src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/joanna.webp"
                    />
                    <div
                      class="MuiCardContent-root testimonials-card-content"
                    >
                      <p
                        class="MuiTypography-root testimonials-card-title MuiTypography-body1"
                      >
                        Joanna
                      </p>
                      <p
                        class="MuiTypography-root testimonials-card-subheader MuiTypography-body1"
                      >
                        Senior Digital Tax Specialist
                      </p>
                      <div
                        class="MuiCollapse-root"
                        style="min-height: 120px;"
                      >
                        <div
                          class="MuiCollapse-wrapper"
                        >
                          <div
                            class="MuiCollapse-wrapperInner"
                          >
                            <p
                              class="MuiTypography-root testimonials-card-text MuiTypography-body1"
                            >
                              Joe is very motivated...
                            </p>
                          </div>
                        </div>
                      </div>
                      <button
                        aria-expanded="false"
                        aria-label="show more"
                        class="MuiButtonBase-root MuiIconButton-root testimonials-expand"
                        tabindex="0"
                        type="button"
                      >
                        <span
                          class="MuiIconButton-label"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                            />
                          </svg>
                        </span>
                        <span
                          class="MuiTouchRipple-root"
                        />
                      </button>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root testimonials-card-holder MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4"
              >
                <div
                  data-aos="fade-up"
                  data-aos-delay="150"
                >
                  <div
                    class="MuiPaper-root MuiCard-root testimonials-card MuiPaper-elevation3 MuiPaper-rounded"
                  >
                    <img
                      alt="Testimonail photo 2 of 3"
                      class="MuiCardMedia-root testimonials-card-image MuiCardMedia-media MuiCardMedia-img"
                      src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/adama.webp"
                    />
                    <div
                      class="MuiCardContent-root testimonials-card-content"
                    >
                      <p
                        class="MuiTypography-root testimonials-card-title MuiTypography-body1"
                      >
                        Adama
                      </p>
                      <p
                        class="MuiTypography-root testimonials-card-subheader MuiTypography-body1"
                      >
                        Full-stack Developer chez DFDS POLSKA
                      </p>
                      <div
                        class="MuiCollapse-root"
                        style="min-height: 120px;"
                      >
                        <div
                          class="MuiCollapse-wrapper"
                        >
                          <div
                            class="MuiCollapse-wrapperInner"
                          >
                            <p
                              class="MuiTypography-root testimonials-card-text MuiTypography-body1"
                            >
                              Joe have done very good work...
                            </p>
                          </div>
                        </div>
                      </div>
                      <button
                        aria-expanded="false"
                        aria-label="show more"
                        class="MuiButtonBase-root MuiIconButton-root testimonials-expand"
                        tabindex="0"
                        type="button"
                      >
                        <span
                          class="MuiIconButton-label"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                            />
                          </svg>
                        </span>
                      </button>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root testimonials-card-holder MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4"
              >
                <div
                  data-aos="fade-up"
                  data-aos-delay="250"
                >
                  <div
                    class="MuiPaper-root MuiCard-root testimonials-card MuiPaper-elevation3 MuiPaper-rounded"
                  >
                    <img
                      alt="Testimonail photo 3 of 3"
                      class="MuiCardMedia-root testimonials-card-image MuiCardMedia-media MuiCardMedia-img"
                      src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/scott.webp"
                    />
                    <div
                      class="MuiCardContent-root testimonials-card-content"
                    >
                      <p
                        class="MuiTypography-root testimonials-card-title MuiTypography-body1"
                      >
                        Scott
                      </p>
                      <p
                        class="MuiTypography-root testimonials-card-subheader MuiTypography-body1"
                      >
                        BPO
                      </p>
                      <div
                        class="MuiCollapse-root"
                        style="min-height: 120px;"
                      >
                        <div
                          class="MuiCollapse-wrapper"
                        >
                          <div
                            class="MuiCollapse-wrapperInner"
                          >
                            <p
                              class="MuiTypography-root testimonials-card-text MuiTypography-body1"
                            >
                              Joe is one of those rare talents...
                            </p>
                          </div>
                        </div>
                      </div>
                      <button
                        aria-expanded="false"
                        aria-label="show more"
                        class="MuiButtonBase-root MuiIconButton-root testimonials-expand"
                        tabindex="0"
                        type="button"
                      >
                        <span
                          class="MuiIconButton-label"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                            />
                          </svg>
                        </span>
                      </button>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="section "
    >
      <div
        class="MuiContainer-root container-super-wide MuiContainer-maxWidthLg"
      >
        <div
          class="text-centre pt-64 pb-120"
        >
          <div
            data-aos="fade-down"
            data-aos-delay="0"
          >
            <h1
              class="testimonials-caption-text"
            >
              TESTIMONIALS
            </h1>
          </div>
        </div>
        <div
          class="text-centre pb-120"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-6"
          >
            <div
              class="MuiGrid-root testimonials-card-holder MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4"
            >
              <div
                data-aos="fade-up"
                data-aos-delay="350"
              >
                <div
                  class="MuiPaper-root MuiCard-root testimonials-card MuiPaper-elevation0 MuiPaper-rounded"
                >
                  <img
                    alt="Testimonail photo 1 of 3"
                    class="MuiCardMedia-root testimonials-card-image MuiCardMedia-media MuiCardMedia-img"
                    src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/joanna.webp"
                  />
                  <div
                    class="MuiCardContent-root testimonials-card-content"
                  >
                    <p
                      class="MuiTypography-root testimonials-card-title MuiTypography-body1"
                    >
                      Joanna
                    </p>
                    <p
                      class="MuiTypography-root testimonials-card-subheader MuiTypography-body1"
                    >
                      Senior Digital Tax Specialist
                    </p>
                    <div
                      class="MuiCollapse-root"
                      style="min-height: 120px;"
                    >
                      <div
                        class="MuiCollapse-wrapper"
                      >
                        <div
                          class="MuiCollapse-wrapperInner"
                        >
                          <p
                            class="MuiTypography-root testimonials-card-text MuiTypography-body1"
                          >
                            Joe is very motivated...
                          </p>
                        </div>
                      </div>
                    </div>
                    <button
                      aria-expanded="false"
                      aria-label="show more"
                      class="MuiButtonBase-root MuiIconButton-root testimonials-expand"
                      tabindex="0"
                      type="button"
                    >
                      <span
                        class="MuiIconButton-label"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                          />
                        </svg>
                      </span>
                      <span
                        class="MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root testimonials-card-holder MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4"
            >
              <div
                data-aos="fade-up"
                data-aos-delay="150"
              >
                <div
                  class="MuiPaper-root MuiCard-root testimonials-card MuiPaper-elevation3 MuiPaper-rounded"
                >
                  <img
                    alt="Testimonail photo 2 of 3"
                    class="MuiCardMedia-root testimonials-card-image MuiCardMedia-media MuiCardMedia-img"
                    src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/adama.webp"
                  />
                  <div
                    class="MuiCardContent-root testimonials-card-content"
                  >
                    <p
                      class="MuiTypography-root testimonials-card-title MuiTypography-body1"
                    >
                      Adama
                    </p>
                    <p
                      class="MuiTypography-root testimonials-card-subheader MuiTypography-body1"
                    >
                      Full-stack Developer chez DFDS POLSKA
                    </p>
                    <div
                      class="MuiCollapse-root"
                      style="min-height: 120px;"
                    >
                      <div
                        class="MuiCollapse-wrapper"
                      >
                        <div
                          class="MuiCollapse-wrapperInner"
                        >
                          <p
                            class="MuiTypography-root testimonials-card-text MuiTypography-body1"
                          >
                            Joe have done very good work...
                          </p>
                        </div>
                      </div>
                    </div>
                    <button
                      aria-expanded="false"
                      aria-label="show more"
                      class="MuiButtonBase-root MuiIconButton-root testimonials-expand"
                      tabindex="0"
                      type="button"
                    >
                      <span
                        class="MuiIconButton-label"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                          />
                        </svg>
                      </span>
                    </button>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root testimonials-card-holder MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4"
            >
              <div
                data-aos="fade-up"
                data-aos-delay="250"
              >
                <div
                  class="MuiPaper-root MuiCard-root testimonials-card MuiPaper-elevation3 MuiPaper-rounded"
                >
                  <img
                    alt="Testimonail photo 3 of 3"
                    class="MuiCardMedia-root testimonials-card-image MuiCardMedia-media MuiCardMedia-img"
                    src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/testimonials/scott.webp"
                  />
                  <div
                    class="MuiCardContent-root testimonials-card-content"
                  >
                    <p
                      class="MuiTypography-root testimonials-card-title MuiTypography-body1"
                    >
                      Scott
                    </p>
                    <p
                      class="MuiTypography-root testimonials-card-subheader MuiTypography-body1"
                    >
                      BPO
                    </p>
                    <div
                      class="MuiCollapse-root"
                      style="min-height: 120px;"
                    >
                      <div
                        class="MuiCollapse-wrapper"
                      >
                        <div
                          class="MuiCollapse-wrapperInner"
                        >
                          <p
                            class="MuiTypography-root testimonials-card-text MuiTypography-body1"
                          >
                            Joe is one of those rare talents...
                          </p>
                        </div>
                      </div>
                    </div>
                    <button
                      aria-expanded="false"
                      aria-label="show more"
                      class="MuiButtonBase-root MuiIconButton-root testimonials-expand"
                      tabindex="0"
                      type="button"
                    >
                      <span
                        class="MuiIconButton-label"
                      >
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                          />
                        </svg>
                      </span>
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
